src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -1,5 +1,61 @@
 package frc.robot.subsystems;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import frc.robot.Constants.ArmConstants;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
-public class IntakeSubsystem {
-    
+public class IntakeSubsystem extends SubsystemBase{
+  private CANSparkMax leftIntakeMotor, rightIntakeMotor, topIntakeMotor;
```

Comment:
The right and left intake motor are actually also the shooter motors. Could we take this opportunity to rename them?

Reply:
renamed to leftMotor and rightMotor